### ISSUE TITLE
Is there a way to control aspect ratio for hvplot?

### ISSUE THREAD

**Issue by rsignell-usgs** [pyviz/hvplot#75]
I thought from https://github.com/ioam/holoviews/issues/30#issuecomment-104613952 that this might work:
```python
import xarray as xr
import hvplot.xarray
url = 'https://rsignell.s3.amazonaws.com/NLCD/nlcd_2011_landcover_2011_edition_2014_10_10.tif'
da = xr.open_rasterio(url, chunks={'x':2056, 'y':2056})

%%opts Image [width=700 height=500 colorbar=True aspect='equal'] (cmap='magma') 
da[0, 50000:51000, 80000:81000].hvplot()
```
but I guess `aspect` is not a thing.

Is it possible to use `Raster` with hvplot?

I couldn't figure that out either.   😿

**Comment by philippjfr** [comment]
Currently bokeh does not afford us sufficient control over the inner plot dimensions to be able to offer aspect options. There is however a current PR in progress in bokeh that will allow us to take full control over the inner dimensions of a plot and therefore implement reasonable aspect control. I'd suggest checking back once bokeh 1.0 has been released and we've had a chance to expose the appropriate options in holoviews.

**Comment by rsignell-usgs** [comment]
Okay, @philippjfr , thanks.   Is there a way to use `Raster` with hvplot?

**Comment by rsignell-usgs** [comment]
@philippjfr Is this the bokeh PR? https://github.com/bokeh/bokeh/pull/8085#issuecomment-427104927

**Comment by philippjfr** [comment]
That's the one, it's on hold until bokeh 1.1 unfortunately as it's a huge chunk of work and didn't quite make the cut-off for 1.0, which should happen very soon.

**Comment by tomchor** [comment]
@philippjfr I was also trying to set the aspect of a plot and came across this issue. It seems to me that that problem was [fixed on Bokeh's end](https://github.com/bokeh/bokeh/pull/8085) and that it's available now in version 1.1. So I'm checking back to see if this is implemented on `hvplot`'s end.

**Comment by philippjfr** [comment]
It isn't yet but I was planning on spending next weekend on making various updates to hvplot.
